[2025-04-29 01:14:25,617] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 01:14:30,631] INFO basehttp "GET / HTTP/1.1" 301 0
[2025-04-29 01:14:30,745] INFO basehttp code 400, message Bad request version ('\x15\x99\x8f¢+\x06î4]®Ç\x8e\x00ù967Ì8á¯_ú0ÞÊgÔ\x1b!\x14\x98\x03Ë3\x95ÎH\x194Ê%m4%>\x90g')
[2025-04-29 01:14:30,747] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:14:30,750] INFO basehttp code 400, message Bad request version ('úú\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x06\x93êê\x00\x00\x00\x05\x00\x05\x01\x00\x00\x00\x00\x00')
[2025-04-29 01:14:30,751] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:14:32,961] INFO basehttp code 400, message Bad request version ('\x00\x12\x00\x10\x04\x03\x08\x04\x04\x01\x05\x03\x08\x05\x05\x01\x08\x06\x06\x01\x00')
[2025-04-29 01:14:32,961] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:14:32,964] INFO basehttp code 400, message Bad request version ('O\x03c¶¶æÊF\x98hü\x8a²X\x9c\x98r¡wiû\x18ôFwV¹')
[2025-04-29 01:14:32,964] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:14:32,964] INFO basehttp code 400, message Bad request version ("ÁR\x97úõ6ãÒGs1vË\x8cÂþ\x07'\x88\x98Çh¢M/:_&\x93s\x97\x86v\x98\x84\x00m\x80\x81ýÇgx:\x14\x8eDÊ4Ü½¤ô'XWxJU,^ü")
[2025-04-29 01:14:32,964] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:14:32,972] INFO basehttp code 400, message Bad request version ('ÊÊ\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x06súú\x00\x00\x00-\x00\x02\x01\x01\x00+\x00\x07\x06ÚÚ\x03\x04\x03\x03\x00')
[2025-04-29 01:14:32,972] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:14:41,831] INFO basehttp "GET / HTTP/1.1" 301 0
[2025-04-29 01:14:41,831] INFO basehttp code 400, message Bad request version (',î\x18Ü}Ü\x9c+n\x8aõ\x7f\x00')
[2025-04-29 01:14:41,831] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:14:41,831] INFO basehttp code 400, message Bad request version ('\x9a\x9a\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x06\x93')
[2025-04-29 01:14:41,831] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:14:43,501] INFO basehttp code 400, message Bad request version ('Q1\x12%î9!þ\x024Ø\x02np')
[2025-04-29 01:14:43,502] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:14:43,504] INFO basehttp code 400, message Bad request version ('¡¸@\x84PÙ\x96Ä\x1a:Õl¿µÆÉ\x81\x86\x18Ç×\x99\x9c\x19\x86')
[2025-04-29 01:14:43,504] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:13,985] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_checker_project\settings.py changed, reloading.
[2025-04-29 01:15:14,906] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 01:15:18,458] INFO basehttp code 400, message Bad request version ('ªª\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x063úú\x00\x00\x00\x17\x00\x00\x00+\x00\x07\x06ÊÊ\x03\x04\x03\x03\x00-\x00\x02\x01\x01\x00')
[2025-04-29 01:15:18,458] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:18,458] INFO basehttp code 400, message Bad request version ('RÎ¬º\x81\xadÌªi\x82¥½élb¤¿\x14JP½\x84')
[2025-04-29 01:15:18,458] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:24,782] INFO basehttp code 400, message Bad request version ("Å\x07\x8c;^Hk'Ô\x99i")
[2025-04-29 01:15:24,782] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:24,790] INFO basehttp code 400, message Bad request version ('únå\x88\x83\x9f')
[2025-04-29 01:15:24,791] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:31,121] INFO basehttp code 400, message Bad request version ("C:\x18msk'þ¹")
[2025-04-29 01:15:31,121] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:31,121] INFO basehttp code 400, message Bad request version ('\x00')
[2025-04-29 01:15:31,121] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:39,734] INFO basehttp code 400, message Bad request version ('\x81\x99Ö\x97\x08¦H;Úqcm{jðÖ\x19Tÿ\x8aêá\x10\x18Ð9\x9e\x11[·¶\x8dXs\x90\x88²iýª*>°£K\x8eLâ')
[2025-04-29 01:15:39,736] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:39,736] INFO basehttp code 400, message Bad request version ('KÖÜâ')
[2025-04-29 01:15:39,738] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:39,742] INFO basehttp code 400, message Bad request version ('cÔ´æ')
[2025-04-29 01:15:39,742] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:39,743] INFO basehttp code 400, message Bad request version ('_')
[2025-04-29 01:15:39,743] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:48,686] INFO basehttp code 400, message Bad request syntax ('\x16\x03\x01\x06à\x01\x00\x06Ü\x03\x03{\x126z\x17NÓÚ\x92ç')
[2025-04-29 01:15:48,686] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:48,690] INFO basehttp code 400, message Bad request syntax ('\x16\x03\x01\x07\x00\x01\x00\x06ü\x03\x03m\x8b¯\x9d±\x1b\x05ÙßU®ú')
[2025-04-29 01:15:48,690] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:53,799] INFO basehttp code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x06')
[2025-04-29 01:15:53,799] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:53,803] INFO basehttp code 400, message Bad request version ('Ï\x05dtw[§')
[2025-04-29 01:15:53,804] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:54,440] INFO basehttp code 400, message Bad request version ('ZZ\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x06sÊÊ\x00\x00\x00')
[2025-04-29 01:15:54,440] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:54,443] INFO basehttp code 400, message Bad request version ('·;g\x8dù\x92n<Û3ÝÏÿ¥\x03¹p}$')
[2025-04-29 01:15:54,443] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:54,665] INFO basehttp code 400, message Bad request version ('\x00ú\x00\x00\x01\x00\x01p\x00')
[2025-04-29 01:15:54,666] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:54,670] INFO basehttp code 400, message Bad request version ('\x15¿8²Hü\x92ß')
[2025-04-29 01:15:54,670] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:54,840] INFO basehttp code 400, message Bad request version ('ºº\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x063zz\x00\x00\x00')
[2025-04-29 01:15:54,840] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:54,843] INFO basehttp code 400, message Bad request version ('ZZ\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x06\x93ªª\x00\x00\x00')
[2025-04-29 01:15:54,844] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:55,011] INFO basehttp code 400, message Bad request version ('ùÂ\x00\x80\x94Sqe°\x932¶¬\\\x1bZ¢Y')
[2025-04-29 01:15:55,011] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:15:55,014] INFO basehttp code 400, message Bad request version ('ºº\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x06súú\x00\x00\x00-\x00\x02\x01\x01\x00')
[2025-04-29 01:15:55,014] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:16:24,904] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 01:16:27,526] INFO basehttp code 400, message Bad request version ('**\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x06SJJ\x00\x00\x00')
[2025-04-29 01:16:27,527] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:16:27,531] INFO basehttp code 400, message Bad request version (":\x87OÂ¹Y²?\x06\x8cV·ÀÈ\x8b¢\x92x\x99Î¾'·æ\x1a\x86ç\x137bü«J#97¤7uÉ¢\x931L(y_«Ö\x03Î¨0XÛ`jj«>ÈT/0\x9a±ô\x86AÌT")
[2025-04-29 01:16:27,532] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:18:06,407] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_checker_project\settings.py changed, reloading.
[2025-04-29 01:18:08,579] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 01:18:12,859] INFO basehttp code 400, message Bad request version ('â§\x89³W;C\x89R=ºCb')
[2025-04-29 01:18:12,860] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:18:12,864] INFO basehttp code 400, message Bad request version ('Å_\x9aÀ\x05ë\x97l(')
[2025-04-29 01:18:12,865] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:18:19,690] INFO basehttp code 400, message Bad request version ('\x02h2\x08http/1.1DÍ\x00\x05\x00\x03\x02h2\x00')
[2025-04-29 01:18:19,690] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:18:19,690] INFO basehttp code 400, message Bad request version ('\x02h2\x08http/1.1\x00')
[2025-04-29 01:18:19,690] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:19:13,761] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 01:19:22,442] INFO basehttp "GET / HTTP/1.1" 200 176508
[2025-04-29 01:19:22,796] INFO basehttp "GET /static/ip_app/css/tailwind.css HTTP/1.1" 200 7026
[2025-04-29 01:19:22,798] INFO basehttp "GET /static/ip_app/css/styles.css HTTP/1.1" 200 64267
[2025-04-29 01:19:24,602] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 200 33591
[2025-04-29 01:19:28,843] INFO basehttp "GET /static/ip_app/img/favicon/site.webmanifest HTTP/1.1" 200 441
[2025-04-29 01:19:28,893] WARNING log Not Found: /favicon.ico
[2025-04-29 01:19:28,893] WARNING basehttp "GET /favicon.ico HTTP/1.1" 404 4669
[2025-04-29 01:19:28,908] WARNING log Not Found: /favicon.svg
[2025-04-29 01:19:28,908] WARNING basehttp "GET /favicon.svg HTTP/1.1" 404 4669
[2025-04-29 01:19:28,925] WARNING basehttp "GET /static/ip_app/img/favicon/favicon-32x32.png HTTP/1.1" 404 1922
[2025-04-29 01:19:28,942] WARNING basehttp "GET /static/ip_app/img/favicon/favicon-16x16.png HTTP/1.1" 404 1922
[2025-04-29 01:19:28,960] WARNING log Not Found: /favicon-96x96.png
[2025-04-29 01:19:28,960] WARNING basehttp "GET /favicon-96x96.png HTTP/1.1" 404 4687
[2025-04-29 01:19:44,475] INFO basehttp "GET /ip-lookup HTTP/1.1" 301 0
[2025-04-29 01:19:49,455] INFO basehttp "GET /ip-lookup/ HTTP/1.1" 200 127443
[2025-04-29 01:20:00,222] INFO basehttp "GET / HTTP/1.1" 200 176508
[2025-04-29 01:20:05,260] INFO basehttp "GET /ip-lookup?ip=103.125.235.21 HTTP/1.1" 301 0
[2025-04-29 01:20:08,757] INFO basehttp "GET /ip-lookup/?ip=103.125.235.21 HTTP/1.1" 200 127421
[2025-04-29 01:23:05,389] INFO basehttp code 400, message Bad request version ('ñ')
[2025-04-29 01:23:05,395] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:23:05,399] INFO basehttp code 400, message Bad request version ('jj\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x06S')
[2025-04-29 01:23:05,399] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:23:07,547] INFO basehttp code 400, message Bad request version (":Z$z°»Õä\x00¾¦Æ'ä\x1aÿ£K")
[2025-04-29 01:23:07,547] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:23:07,551] INFO basehttp code 400, message Bad request version ('=@\x05\x9e\x8b¨k\x8e[æI\x86×}')
[2025-04-29 01:23:07,551] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:23:16,200] INFO basehttp code 400, message Bad request version ('RüÑ\x0638v\x8dd\x8bhö³\x91\x8a7H\x9a6\x9c(\x8aô\x95\xad$')
[2025-04-29 01:23:16,201] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:23:16,201] INFO basehttp code 400, message Bad request version ('**\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x063\x9a\x9a\x00\x00\x00')
[2025-04-29 01:23:16,202] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:23:16,207] INFO basehttp code 400, message Bad request version ('\x00\x02\x01\x00\x00#\x00\x00\x00')
[2025-04-29 01:23:16,208] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:23:16,208] INFO basehttp code 400, message Bad request version ('¹')
[2025-04-29 01:23:16,209] ERROR basehttp You're accessing the development server over HTTPS, but it only supports HTTP.
[2025-04-29 01:23:25,345] INFO basehttp "GET / HTTP/1.1" 200 176508
[2025-04-29 01:23:25,413] INFO basehttp "GET /static/ip_app/css/tailwind.css HTTP/1.1" 200 7026
[2025-04-29 01:23:25,415] INFO basehttp "GET /static/ip_app/css/styles.css HTTP/1.1" 200 64267
[2025-04-29 01:23:26,976] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 200 33591
[2025-04-29 01:23:31,664] WARNING log Not Found: /favicon.ico
[2025-04-29 01:23:31,666] WARNING basehttp "GET /favicon.ico HTTP/1.1" 404 4669
[2025-04-29 01:23:31,743] WARNING log Not Found: /favicon.svg
[2025-04-29 01:23:31,746] WARNING basehttp "GET /favicon.svg HTTP/1.1" 404 4669
[2025-04-29 01:23:31,763] WARNING basehttp "GET /static/ip_app/img/favicon/favicon-32x32.png HTTP/1.1" 404 1922
[2025-04-29 01:23:31,777] WARNING basehttp "GET /static/ip_app/img/favicon/favicon-16x16.png HTTP/1.1" 404 1922
[2025-04-29 01:23:31,779] WARNING log Not Found: /favicon-96x96.png
[2025-04-29 01:23:31,794] WARNING basehttp "GET /favicon-96x96.png HTTP/1.1" 404 4687
[2025-04-29 01:27:43,607] INFO basehttp "GET / HTTP/1.1" 200 177692
[2025-04-29 01:27:43,657] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 304 0
[2025-04-29 01:27:56,144] INFO basehttp "GET / HTTP/1.1" 200 177692
[2025-04-29 01:28:30,785] INFO basehttp "GET / HTTP/1.1" 200 176347
[2025-04-29 01:29:00,268] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 01:29:08,666] INFO basehttp "GET / HTTP/1.1" 200 176347
[2025-04-29 01:29:08,768] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 304 0
[2025-04-29 01:29:19,567] INFO basehttp "GET /contact HTTP/1.1" 301 0
[2025-04-29 01:29:19,787] INFO basehttp "GET /contact/ HTTP/1.1" 200 29168
[2025-04-29 01:29:19,856] WARNING basehttp "GET /static/ip_app/css/style.css HTTP/1.1" 404 1874
[2025-04-29 01:30:39,591] INFO basehttp "GET / HTTP/1.1" 200 176202
[2025-04-29 01:30:39,627] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 304 0
[2025-04-29 01:30:41,549] WARNING log Not Found: /my-ip
[2025-04-29 01:30:41,549] WARNING basehttp "GET /my-ip HTTP/1.1" 404 4651
[2025-04-29 01:31:12,185] INFO basehttp "GET / HTTP/1.1" 200 176206
[2025-04-29 01:31:14,553] INFO basehttp "GET /ip-lookup HTTP/1.1" 301 0
[2025-04-29 01:31:19,762] INFO basehttp "GET /ip-lookup/ HTTP/1.1" 200 127141
[2025-04-29 01:31:48,780] INFO basehttp - Broken pipe from ('127.0.0.1', 50960)
[2025-04-29 01:41:20,741] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 01:48:07,005] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 01:48:24,854] INFO basehttp "GET / HTTP/1.1" 200 177824
[2025-04-29 01:48:25,324] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 200 16968
[2025-04-29 01:49:02,701] INFO basehttp "GET /ip-lookup/ HTTP/1.1" 200 128759
[2025-04-29 01:49:02,770] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 304 0
[2025-04-29 01:49:36,794] INFO basehttp "GET /contact/ HTTP/1.1" 200 30645
[2025-04-29 01:49:36,868] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 304 0
[2025-04-29 01:49:36,869] WARNING basehttp "GET /static/ip_app/css/style.css HTTP/1.1" 404 1874
[2025-04-29 01:49:43,467] INFO basehttp - Broken pipe from ('127.0.0.1', 54213)
[2025-04-29 01:49:44,009] INFO basehttp - Broken pipe from ('127.0.0.1', 54214)
[2025-04-29 01:49:44,567] INFO basehttp "GET / HTTP/1.1" 200 177824
[2025-04-29 01:50:44,113] INFO basehttp "GET / HTTP/1.1" 200 176055
[2025-04-29 01:50:44,166] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 304 0
[2025-04-29 02:24:54,856] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 02:25:07,373] INFO basehttp "GET / HTTP/1.1" 200 220683
[2025-04-29 02:25:07,518] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 304 0
[2025-04-29 02:26:00,681] WARNING log Not Found: /blog
[2025-04-29 02:26:00,685] WARNING basehttp "GET /blog HTTP/1.1" 404 4648
[2025-04-29 02:26:23,148] INFO basehttp "GET /contact/ HTTP/1.1" 200 28876
[2025-04-29 02:26:23,218] WARNING basehttp "GET /static/ip_app/css/style.css HTTP/1.1" 404 1874
[2025-04-29 02:26:27,830] WARNING log Not Found: /status
[2025-04-29 02:26:27,831] WARNING basehttp "GET /status HTTP/1.1" 404 4654
[2025-04-29 02:26:36,461] INFO basehttp "GET / HTTP/1.1" 200 176055
[2025-04-29 08:32:37,026] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 08:32:46,660] INFO basehttp "GET / HTTP/1.1" 200 176105
[2025-04-29 08:32:46,855] INFO basehttp "GET /static/ip_app/css/tailwind.css HTTP/1.1" 304 0
[2025-04-29 08:32:46,910] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 304 0
[2025-04-29 08:32:47,336] INFO basehttp "GET /static/ip_app/img/favicon/site.webmanifest HTTP/1.1" 304 0
[2025-04-29 08:35:35,067] INFO basehttp "GET / HTTP/1.1" 200 175730
[2025-04-29 08:36:56,073] INFO basehttp "GET / HTTP/1.1" 200 175733
[2025-04-29 08:37:07,812] INFO basehttp "GET /ip-lookup/ HTTP/1.1" 200 126618
[2025-04-29 08:37:43,781] INFO basehttp "GET / HTTP/1.1" 200 175737
[2025-04-29 08:42:10,633] INFO basehttp "GET / HTTP/1.1" 200 175723
[2025-04-29 08:42:37,732] INFO basehttp "GET / HTTP/1.1" 200 175723
[2025-04-29 08:42:38,080] INFO basehttp "GET /static/ip_app/css/tailwind.css HTTP/1.1" 200 7026
[2025-04-29 08:42:38,089] INFO basehttp "GET /static/ip_app/css/styles.css HTTP/1.1" 200 64267
[2025-04-29 08:42:39,903] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 200 16968
[2025-04-29 08:50:27,141] INFO basehttp "GET / HTTP/1.1" 200 167697
[2025-04-29 08:53:43,230] INFO basehttp "GET / HTTP/1.1" 200 177858
[2025-04-29 08:54:18,606] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 08:54:32,967] INFO basehttp "GET / HTTP/1.1" 200 177858
[2025-04-29 09:03:50,803] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'else'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 1178: 'else', expected 'endblock'. Did you forget to register or load this tag?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'else'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 1178: 'else', expected 'endblock'. Did you forget to register or load this tag?
[2025-04-29 09:03:50,835] ERROR basehttp "GET / HTTP/1.1" 500 335867
[2025-04-29 09:05:00,985] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'else'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 1176: 'else', expected 'endblock'. Did you forget to register or load this tag?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'else'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 1176: 'else', expected 'endblock'. Did you forget to register or load this tag?
[2025-04-29 09:05:00,994] ERROR basehttp "GET / HTTP/1.1" 500 335867
[2025-04-29 09:05:45,774] INFO basehttp "GET / HTTP/1.1" 200 175678
[2025-04-29 09:06:32,862] INFO basehttp "GET / HTTP/1.1" 200 175152
[2025-04-29 09:07:06,086] INFO basehttp "GET / HTTP/1.1" 200 175152
[2025-04-29 09:08:07,037] INFO basehttp "GET / HTTP/1.1" 200 175748
[2025-04-29 09:08:41,133] INFO basehttp "GET / HTTP/1.1" 200 175165
[2025-04-29 09:10:54,652] INFO basehttp "GET / HTTP/1.1" 200 174976
[2025-04-29 09:13:24,464] INFO basehttp "GET / HTTP/1.1" 200 175181
[2025-04-29 09:13:40,464] INFO basehttp "GET / HTTP/1.1" 200 175181
[2025-04-29 09:16:33,602] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:16:33,724] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 304 0
[2025-04-29 09:17:16,463] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:17:36,617] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:17:50,593] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:17:56,425] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:19:31,503] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:19:45,231] INFO basehttp "GET / HTTP/1.1" 200 175319
[2025-04-29 09:20:00,200] INFO basehttp "GET / HTTP/1.1" 200 175319
[2025-04-29 09:20:06,245] INFO basehttp "GET / HTTP/1.1" 200 175319
[2025-04-29 09:20:14,457] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:22:01,459] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:22:07,551] INFO basehttp - Broken pipe from ('127.0.0.1', 61544)
[2025-04-29 09:22:17,575] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:22:31,256] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:22:45,467] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:22:47,378] INFO basehttp "GET /ip-lookup HTTP/1.1" 301 0
[2025-04-29 09:22:52,555] INFO basehttp "GET /ip-lookup/ HTTP/1.1" 200 126636
[2025-04-29 09:23:01,885] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:23:35,942] INFO basehttp "GET / HTTP/1.1" 200 175320
[2025-04-29 09:42:22,194] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_checker_project\settings.py changed, reloading.
[2025-04-29 09:42:24,573] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 10:01:41,778] INFO basehttp "GET / HTTP/1.1" 200 76119
[2025-04-29 10:01:41,929] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:01:41,935] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:03:48,883] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 228, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'extra_css' appears more than once

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 228, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'extra_css' appears more than once
[2025-04-29 10:03:48,898] ERROR basehttp "GET / HTTP/1.1" 500 371263
[2025-04-29 10:06:58,462] INFO basehttp "GET / HTTP/1.1" 200 176553
[2025-04-29 10:06:58,523] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:06:58,524] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:07:22,364] INFO basehttp "GET / HTTP/1.1" 200 176553
[2025-04-29 10:07:22,447] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:07:22,454] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:07:40,429] INFO basehttp "GET / HTTP/1.1" 200 177021
[2025-04-29 10:07:40,482] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:07:40,488] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:07:45,705] INFO basehttp "GET / HTTP/1.1" 200 177021
[2025-04-29 10:07:45,788] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:07:45,796] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:08:01,036] INFO basehttp "GET / HTTP/1.1" 200 175770
[2025-04-29 10:08:01,098] INFO basehttp "GET /static/ip_app/js/main.js HTTP/1.1" 304 0
[2025-04-29 10:11:25,982] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 228, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'extra_meta' appears more than once

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 228, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'extra_meta' appears more than once
[2025-04-29 10:11:26,011] ERROR basehttp "GET / HTTP/1.1" 500 313039
[2025-04-29 10:11:50,477] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 228, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'extra_meta' appears more than once

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 228, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'extra_meta' appears more than once
[2025-04-29 10:11:50,483] ERROR basehttp "GET / HTTP/1.1" 500 312579
[2025-04-29 10:12:05,195] INFO basehttp "GET / HTTP/1.1" 200 178742
[2025-04-29 10:12:05,261] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:12:05,262] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:12:21,568] INFO basehttp "GET / HTTP/1.1" 200 178742
[2025-04-29 10:12:21,664] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:12:21,677] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:12:36,480] INFO basehttp "GET / HTTP/1.1" 200 175770
[2025-04-29 10:13:24,434] INFO basehttp "GET / HTTP/1.1" 200 177082
[2025-04-29 10:13:24,644] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:13:24,649] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:13:35,995] INFO basehttp "GET / HTTP/1.1" 200 177082
[2025-04-29 10:13:36,098] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:13:36,099] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:15:26,580] INFO basehttp "GET / HTTP/1.1" 200 177507
[2025-04-29 10:15:26,691] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:15:26,691] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:16:52,657] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'endblock'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 577, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 84: 'endblock'. Did you forget to register or load this tag?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'endblock'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 577, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 84: 'endblock'. Did you forget to register or load this tag?
[2025-04-29 10:16:52,689] ERROR basehttp "GET / HTTP/1.1" 500 371540
[2025-04-29 10:18:35,015] INFO basehttp "GET / HTTP/1.1" 200 178147
[2025-04-29 10:18:35,081] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:18:35,082] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:18:56,078] INFO basehttp "GET / HTTP/1.1" 200 178180
[2025-04-29 10:18:56,142] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:18:56,147] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:23:23,176] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 10:23:31,545] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 577, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 76: 'compress'. Did you forget to register or load this tag?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 1016, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 977, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 577, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 76: 'compress'. Did you forget to register or load this tag?
[2025-04-29 10:23:31,553] ERROR basehttp "GET / HTTP/1.1" 500 313352
[2025-04-29 10:24:02,307] INFO basehttp - Broken pipe from ('127.0.0.1', 64332)
[2025-04-29 10:24:06,083] INFO basehttp "GET / HTTP/1.1" 200 178214
[2025-04-29 10:24:06,211] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:24:06,211] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:24:06,416] INFO basehttp "GET /static/compressed/css/output.0085a4dbb7f4.css HTTP/1.1" 200 49991
[2025-04-29 10:34:00,657] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?
[2025-04-29 10:34:00,858] ERROR basehttp "GET / HTTP/1.1" 500 323915
[2025-04-29 10:34:17,686] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?
[2025-04-29 10:34:17,692] ERROR basehttp "GET / HTTP/1.1" 500 323915
[2025-04-29 10:34:29,852] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?
[2025-04-29 10:34:29,854] ERROR basehttp "GET / HTTP/1.1" 500 323915
[2025-04-29 10:34:51,367] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 10:34:57,679] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?
[2025-04-29 10:34:57,691] ERROR basehttp "GET / HTTP/1.1" 500 323778
[2025-04-29 10:35:27,114] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?
[2025-04-29 10:35:27,114] ERROR basehttp "GET / HTTP/1.1" 500 323915
[2025-04-29 10:36:25,684] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?
[2025-04-29 10:36:25,690] ERROR basehttp "GET / HTTP/1.1" 500 323915
[2025-04-29 10:38:19,603] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 10:38:26,263] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'compress'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 3411: 'compress', expected 'endblock'. Did you forget to register or load this tag?
[2025-04-29 10:38:26,279] ERROR basehttp "GET / HTTP/1.1" 500 323778
[2025-04-29 10:39:41,690] INFO basehttp "GET / HTTP/1.1" 200 178232
[2025-04-29 10:39:41,840] INFO basehttp "GET /static/compressed/css/output.0085a4dbb7f4.css HTTP/1.1" 304 0
[2025-04-29 10:39:41,847] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:39:41,848] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:39:51,489] INFO basehttp "GET / HTTP/1.1" 200 178232
[2025-04-29 10:39:51,780] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:39:51,787] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:40:24,187] INFO basehttp "GET / HTTP/1.1" 200 177971
[2025-04-29 10:40:24,271] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:40:24,279] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:40:28,100] INFO basehttp "GET / HTTP/1.1" 200 177971
[2025-04-29 10:40:28,184] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:40:28,193] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:40:42,056] INFO basehttp "GET / HTTP/1.1" 200 177971
[2025-04-29 10:40:42,216] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 10:40:42,247] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 10:40:42,492] INFO basehttp "GET /static/compressed/css/output.0085a4dbb7f4.css HTTP/1.1" 200 49991
[2025-04-29 10:44:30,292] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_checker_project\settings.py changed, reloading.
[2025-04-29 10:44:32,031] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 10:45:23,708] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_checker_project\settings.py changed, reloading.
[2025-04-29 10:45:25,013] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 10:50:41,060] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py changed, reloading.
[2025-04-29 10:50:42,919] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 10:51:18,791] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py changed, reloading.
[2025-04-29 10:51:19,627] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 10:52:44,638] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py changed, reloading.
[2025-04-29 10:52:46,020] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 10:57:29,377] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py changed, reloading.
[2025-04-29 10:57:31,702] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 10:58:21,371] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py changed, reloading.
[2025-04-29 10:58:22,589] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 11:00:27,921] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 228, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'title' appears more than once

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 228, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'title' appears more than once
[2025-04-29 11:00:27,968] ERROR basehttp "GET / HTTP/1.1" 500 31944
[2025-04-29 11:00:52,821] ERROR log Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 35, in index
    return render(request, 'ip_app/index.html', {'ip_data': ip_data})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 228, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'title' appears more than once

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_app\views.py", line 40, in index
    return render(request, 'ip_app/index.html', {'ip_data': {'query': 'Error loading', 'error': str(e)}})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\manik\AppData\Roaming\Python\Python312\site-packages\django\template\loader_tags.py", line 228, in do_block
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'block' tag with name 'title' appears more than once
[2025-04-29 11:00:52,832] ERROR basehttp "GET / HTTP/1.1" 500 31891
[2025-04-29 11:03:29,452] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_checker_project\settings.py changed, reloading.
[2025-04-29 11:08:33,708] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 11:10:21,919] INFO autoreload C:\Users\manik\OneDrive\Desktop\i\ip_checker_project\ip_checker_project\settings.py changed, reloading.
[2025-04-29 11:10:23,077] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 11:11:34,196] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 11:11:40,819] INFO basehttp "GET / HTTP/1.1" 200 35377
[2025-04-29 11:11:40,918] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:11:40,924] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:11:40,934] INFO basehttp "GET /static/compressed/css/output.0085a4dbb7f4.css HTTP/1.1" 304 0
[2025-04-29 11:11:54,299] INFO basehttp "GET / HTTP/1.1" 200 35318
[2025-04-29 11:11:54,364] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:11:54,383] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:11:54,383] INFO basehttp "GET /static/compressed/css/output.0085a4dbb7f4.css HTTP/1.1" 304 0
[2025-04-29 11:12:45,144] INFO basehttp "GET / HTTP/1.1" 200 35305
[2025-04-29 11:12:45,246] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:12:45,268] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:12:45,616] INFO basehttp "GET /static/compressed/css/output.0085a4dbb7f4.css HTTP/1.1" 200 49991
[2025-04-29 11:12:46,454] INFO basehttp "GET /static/ip_app/img/favicon/site.webmanifest HTTP/1.1" 200 441
[2025-04-29 11:12:59,724] INFO basehttp "GET / HTTP/1.1" 200 35305
[2025-04-29 11:12:59,859] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:12:59,871] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:13:51,124] INFO basehttp "GET / HTTP/1.1" 200 35380
[2025-04-29 11:13:51,259] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:13:51,268] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:14:07,594] INFO basehttp "GET / HTTP/1.1" 200 35333
[2025-04-29 11:14:07,689] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:14:07,701] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:15:02,876] INFO basehttp "GET / HTTP/1.1" 200 35362
[2025-04-29 11:15:03,016] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:15:03,016] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:16:01,336] INFO basehttp "GET / HTTP/1.1" 200 35107
[2025-04-29 11:16:01,468] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:16:01,485] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:16:17,085] INFO basehttp "GET / HTTP/1.1" 200 35082
[2025-04-29 11:16:17,213] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:16:17,218] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:16:29,566] INFO basehttp "GET / HTTP/1.1" 200 35141
[2025-04-29 11:16:29,706] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:16:29,721] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:16:43,632] INFO basehttp "GET / HTTP/1.1" 200 35168
[2025-04-29 11:16:43,785] INFO basehttp "GET /static/ip_app/css/tailwind.css HTTP/1.1" 200 7026
[2025-04-29 11:16:43,797] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:16:43,816] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:17:07,811] INFO basehttp "GET / HTTP/1.1" 200 35082
[2025-04-29 11:17:07,895] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:17:07,910] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:18:09,220] INFO basehttp - Broken pipe from ('127.0.0.1', 51440)
[2025-04-29 11:18:15,909] INFO basehttp "GET / HTTP/1.1" 200 35331
[2025-04-29 11:18:16,152] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:18:16,182] INFO basehttp "GET /static/compressed/css/output.0085a4dbb7f4.css HTTP/1.1" 304 0
[2025-04-29 11:18:16,184] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:18:35,939] INFO basehttp "GET / HTTP/1.1" 200 35379
[2025-04-29 11:18:36,128] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:18:36,128] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:18:56,179] INFO basehttp "GET / HTTP/1.1" 200 35333
[2025-04-29 11:18:56,253] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:18:56,253] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:19:17,702] INFO basehttp "GET / HTTP/1.1" 200 35293
[2025-04-29 11:19:17,783] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:19:17,793] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 11:19:30,754] INFO basehttp "GET / HTTP/1.1" 200 35296
[2025-04-29 11:19:30,835] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 11:19:30,849] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 23:30:06,923] INFO autoreload Watching for file changes with StatReloader
[2025-04-29 23:30:20,009] INFO basehttp "GET / HTTP/1.1" 200 35332
[2025-04-29 23:30:20,142] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 23:30:20,150] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 23:30:20,287] INFO basehttp "GET /static/compressed/css/output.0085a4dbb7f4.css HTTP/1.1" 200 49991
[2025-04-29 23:30:20,287] INFO basehttp "GET /static/compressed/js/output.5d5c72a9f12e.js HTTP/1.1" 200 10263
[2025-04-29 23:30:23,189] WARNING basehttp "GET /static/ip_app/img/favicon/favicon-32x32.png HTTP/1.1" 404 1922
[2025-04-29 23:30:23,211] INFO basehttp "GET /static/ip_app/img/favicon/site.webmanifest HTTP/1.1" 200 441
[2025-04-29 23:30:23,243] WARNING basehttp "GET /static/ip_app/img/favicon/favicon-16x16.png HTTP/1.1" 404 1922
[2025-04-29 23:30:42,784] INFO basehttp "GET / HTTP/1.1" 200 35362
[2025-04-29 23:30:42,934] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
[2025-04-29 23:30:42,968] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 23:30:42,986] INFO basehttp "GET /static/compressed/css/output.0085a4dbb7f4.css HTTP/1.1" 200 49991
[2025-04-29 23:30:43,012] INFO basehttp "GET /static/compressed/js/output.5d5c72a9f12e.js HTTP/1.1" 200 10263
[2025-04-29 23:31:40,172] INFO basehttp "GET /ip-lookup HTTP/1.1" 301 0
[2025-04-29 23:31:44,849] INFO basehttp "GET /ip-lookup/ HTTP/1.1" 200 24018
[2025-04-29 23:31:44,926] WARNING basehttp "GET /static/ip_app/js/main.min.js HTTP/1.1" 404 1877
[2025-04-29 23:31:44,926] WARNING basehttp "GET /static/ip_app/css/styles.min.css HTTP/1.1" 404 1889
